opm/common/ErrorMacros.hpp

```diff
@@ -48,23 +47,25 @@
 // exception class derived from either std::logic_error or
 // std::runtime_error.
 //
-// Usage: OPM_THROW(ExceptionClass, "Error message " << value);
-#define OPM_THROW(Exception, message)                                                        \
-    do {                                                                                     \
-        std::ostringstream opmErrorMacroOStringStream;                                       \
-        opmErrorMacroOStringStream << "[" << __FILE__ << ":" << __LINE__ << "] " << message; \
-        ::Opm::OpmLog::error(opmErrorMacroOStringStream.str());                              \
-        throw Exception(opmErrorMacroOStringStream.str());                                   \
+// Usage: OPM_THROW(ExceptionClass, "Error message");
+#define OPM_THROW(Exception, message)                           \
+    do {                                                        \
+        std::string oss__ = std::string{"["} + __FILE__ + ":" + \
```

Comment:
Tiny nit: Per the C++ standard, user code is not allowed to use names that have two consecutive underscores **anywhere**.  Those symbols are reserved for the implementation, so `oss__` isn't really allowed here.  I've certainly violated that constraint myself, not least in an earlier version of this macro, but now I try really hard to avoid doing that.

Reply:
removed one underscore.